docs/didcomm_router.md

```diff
@@ -0,0 +1,32 @@
+# Aries DIDComm Router/Mediator
+
+The aries-framework-go project can be used a DIDComm Router/Mediator. The agents that do not 
+have inbound transport can register with the router. Basically, the agent asks another agent to 
+route the messages to it by asking the permission. On successful grant, agent receives the 
```

Comment:
for permission.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/523808246463b15b8a0e880d8cbc9cb889558772..3fe47f63bb2fb52c165715cd5072a3a12a272d8f).